src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheckTest.java

```diff
@@ -105,6 +105,7 @@ public void testAlone() throws Exception {
             "184:9: " + getCheckMessage(MSG_KEY_LINE_ALONE, "}", 9),
             "189:9: " + getCheckMessage(MSG_KEY_LINE_ALONE, "}", 9),
             "190:53: " + getCheckMessage(MSG_KEY_LINE_ALONE, "}", 53),
+            "221:12: " + getCheckMessage(MSG_KEY_LINE_ALONE, "}", 12),
```

Comment:
@arinmodi please create new input file; we are trying to keep input files shorter than 120 lines: https://github.com/checkstyle/checkstyle/issues/11163. 

To test input, please add some more input form the report, such as https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/1cb9b86_2022085214/reports/diff/sevntu-checkstyle/xref/home/runner/work/checkstyle/checkstyle/contribution/checkstyle-tester/repositories/sevntu-checkstyle/sevntu-checks/src/test/resources/com/github/sevntu/checkstyle/checks/coding/InputAvoidHidingCauseExceptionCheck.java.html#L71,  and a bunch from https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/1cb9b86_2022085214/reports/diff/sevntu-checkstyle/index.html#A54

Reply:
Done